Makefile

```diff
@@ -105,7 +105,7 @@ docker-test-unit: $(DOCKER_CACHE)
 .PHONY: docker-test-integration
 docker-test-integration:
 	@echo "===> Building Antrea Integration Test Docker image <==="
-	@docker build -t antrea/test -f build/images/test/Dockerfile .
+	@docker build --pull -t antrea/test -f build/images/test/Dockerfile .
```

Comment:
that may be a bit surprising since a local base image would be overridden, but I imagine few people are likely to be in a situation where they have to test with a local base image

Reply:
Yes, base image is overridden. The output of the make cmd shows that image is being pulled from antrea dockerhub registry; not sure how often people could use a local image. I can add an argument (no-pull) to script, if you think that will be useful.

Reply:
no need, I doubt people will run into this issue